src/client/game.cpp

```diff
@@ -1637,7 +1637,10 @@ bool Game::getServerContent(bool *aborted)
 			std::stringstream message;
 			std::fixed(message);
 			message.precision(0);
-			message << gettext("Media...") << " " << (client->mediaReceiveProgress()*100) << "%";
+			if (client->mediaReceiveProgress() > 0)
+				message << gettext("Media... ") << (client->mediaReceiveProgress() * 100) << "%";
```

Comment:
i think instead you can just condition the % show, here you dup the gettext call

Reply:
What for?

Reply:
He's saying you can do this:

```c++
message.precision(0);
message << gettext("Media... ");
if (client->mediaReceiveProgress() > 0)
	message << (client->mediaReceiveProgress() * 100) << "%";
message.precision(2);
```

Reply:
I understood. But I didn’t think that this was changing anything.
Pls, review changes